<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Guard" name="first.timeout" tests="2" failures="2" timestamp="14 Apr 2020 10:02:06 GMT" time="1.040" errors="0">
  <testcase name="display1" time="0.001" classname="first.timeout"/>
  <testcase name="display" time="1.039" classname="first.timeout">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.display() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.display() didn't finish within the time-out 1000
]]>
    </failure>
  </testcase> <!-- display -->
</testsuite> <!-- first.timeout -->
